### ISSUE TITLE
Notebook 01A: Polynomial, OLS, TF basic regrssion

### ISSUE THREAD

**Issue by abhi8893** [abhi8893/Tensorflow-tutorial-Daniel-Bourke#19]
* Polynomial model
* OLS Regressor
* TF basic regression (loss function optimization)
